Parser/lexer/lexer.c

```diff
@@ -309,7 +309,6 @@ verify_end_of_number(struct tok_state *tok, int c, const char *kind) {
 }
 
 /* Verify that the identifier follows PEP 3131.
-   All identifier strings are guaranteed to be "ready" unicode objects.
```

Comment:
You can put the `*/` from the next line on the same line as `PEP 3131.` as it not fits on a single line:

```c
/* Verify that the identifier follows PEP 3131. */
```